fullstack-cert/html-projects/lab-multimedia-player/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Multimedia Player</title>
+</head>
+<body>
+    <h1>Multimedia Player</h1>
+    <section>
+        <h2>Now Playing: Symphony No. 5</h2>
+        <audio controls aria-label="Audio Player for Symphony No. 5">
+            <source src="symphony-no5.mp3" type="audio/mpeg">
+        </audio>
+    </section>
+
+    <section>
+        <h2>Is AI coming for developer jobs?</h2>
+        <video controls width="300" aria-label="Video Player for Is AI coming for developer jobs?">
+            <source src="Is_AI_coming_for_developer_jobs.mp4" type="video/mp4">
```

Comment:
same issue here.
we should include any mp4 sources in the CDN repo.
But also, I am not sure if we are able to freely use this video. Maybe there is already an mp4 we can use in the CDN repo.
Or better yet, we could use one of the lectures videos mp4s. You can ask one of the lectures stars like Beau or Ania for one of the edited mp4s

Reply:
the video is a short from Quincy's podcast, I have reached out to Beau for one of the lecture videos

Reply:
updated